apps/web/playwright/manage-booking-questions.e2e.ts

```diff
@@ -81,8 +81,8 @@ test.describe("Manage Booking Questions", () => {
         });
 
         await doOnFreshPreview(page, context, async (page) => {
-          const allFieldsLocator = await expectSystemFieldsToBeThereOnBookingPage({ page });
-          const userFieldLocator = allFieldsLocator.nth(5);
```

Comment:
Sometimes this was found on index 4. I don't fully understand why cc @hariombalhara

Reply:
Actually, this approach is not too much reliable for a list of items. It might point to a different element. In fact, Playwright gives a caution about this. https://playwright.dev/docs/locators#get-by-nth-item

Reply:
Agreed. Hence the removal of it 🙏🏽